README.md

```diff
@@ -1,8 +1,8 @@
-## loopback-connector-postgresql
+# loopback-connector-postgresql
 
-The PostgreSQL Connector module for for [loopback-datasource-juggler](http://docs.strongloop.com/loopback-datasource-juggler/).
+The PostgreSQL Connector module for [loopback-datasource-juggler](http://docs.strongloop.com/loopback-datasource-juggler/).
```

Comment:
I think its supposed to connect to the Juggler repo like https://github.com/strongloop/loopback-connector-mysql/blame/master/README.md#L3. However, I think we should just change the entire sentence:

"The official PostgreSQL connector for the LoopBack framework" would suffice. See https://github.com/strongloop/loopback-connector-kv-redis#loopback-connector-kv-redis